src/main/resources/swagger/api-docs.yaml

```diff
@@ -623,7 +673,7 @@ definitions:
           The date and time the cluster was last accessed, in ISO-8601 format.
           Date accessed is defined as the last time the cluster was created, modified, or accessed via the proxy.
       autopauseThreshold:
-        type: int
```

Comment:
This caused an error in [Swagger editor](http://editor.swagger.io). Not sure how it hadn't before.

Reply:
Huh, good find. I usually validate swagger here: https://bigstickcarpet.com/swagger-parser/www/index.html. Maybe there is a difference between validators?

Reply:
That one complains about `int` (vs `integer`) as well.